src/Crud/Controller/CrudControllerTrait.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace Biig\Melodiia\Crud\Controller;
+
+use Biig\Melodiia\Crud\CrudableModelInterface;
+
+trait ControllerTrait
```

Comment:
Naming it CrudControllerTrait would be better. Or ApiControllerTrait

Reply:
I had it in local actually 😂 It's updated